packages/council-cli/src/utils/bigint/removeBigInts.ts

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
Small typo in the filename, it should be `convertBigInts`. But also what do you think of including both sides of the conversion in the name, ie `convertBigIntsToStrings`. This would be more intuitive and convey its purpose more effectively I think.